atom/index.js

```diff
@@ -1,82 +1,94 @@
 import { clean } from '../clean-stores/index.js'
+import { getStoreState, globalContext } from '../context/index.js'
 
-let listenerQueue = []
+let uid = 0
```

Comment:
`uid` is used for `unique id`.

Let’s use `lastId` to explain the it is a counter.